sakura_core/util/file.cpp

```diff
@@ -851,6 +850,7 @@ void GetExistPathW( wchar_t *po , const wchar_t *pi )
 
 	dl = GetExistPath_NO_DriveLetter;	/*「ドライブレターが無い」にしておく*/
 	if( *(po+1)==L':' && WCODE::IsAZ(*po) ){	/* 先頭にドライブレターがある。そのドライブが有効かどうか判定する */
+		wchar_t	drv[4] = L"_:\\";
```

Comment:
```suggestion
		constexpr auto& drv = L"_:\\";
```

Reply:
@berryzplus,
This `drv` should not be `constexpr const` because its first element is changed below the line
```cpp
drv[0] = *po;
```